pjmedia/include/pjmedia/stereo.h

```diff
@@ -129,7 +130,6 @@ PJ_INLINE(pj_status_t) pjmedia_convert_channel_1ton(pj_int16_t multi[],
 
     PJ_ASSERT_RETURN(mono && multi && channel_count && samples_per_frame, 
 		     PJ_EINVAL);
-    PJ_ASSERT_RETURN(options == 0, PJ_EINVAL);
```

Comment:
Use Riza's patch to fix the assertion instead. There's no use in passing non-zero options here anyway.

Reply:
Ok, but this assertion will be removed anyway, if somebody try to make `channel_1ton` and `channel_nto1` fully symmetric, implementing the missing part in `channel_1ton`

say 
- stereo = (mono, mono)
- stereo = (mono, 0)
- stereo = (0, mono)

And when make it, should go and revert Riza's patch.
I think, this will be my second contribution, because I need it. I will open feature request for that, to discuss it.

ps.
Is there an unwritten rule: Don't touch header files. :) 
I know there is some pain in compile time when touching headers, but if no API/ABI changes are introduced all be fine.

Reply:
There's no such rule. It's just that the assertion is not wrong, and it's exactly according to the spec above it (`Options for conversion, currently must be zero`), so there's no reason to remove it. It's the API user that is at fault and that's exactly what the patch fixes.

But if you have plan to extend its features and allow non-zero options, then sure, removing the assertion does make sense.